pkg/util/kube/client.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright The Velero Contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kube
+
+import (
+	"context"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	api "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+)
+
+func PatchRestore(original, updated *api.Restore, kbClient client.Client) error {
```

Comment:
Is this wrapper needed as it is almost the same as the original call?

If we want to keep the wrapper, let's change it to a common way, it should work for any kind of object, not just for `Restore`.

Reply:
Modified. Changed to a more general function.

``` go
func PatchResource(original, updated client.Object, kbClient client.Client) error {
	err := kbClient.Patch(context.Background(), updated, client.MergeFrom(original))

	return err
}
```